sdks/builds/bcl.mk

```diff
@@ -4,17 +4,18 @@
 
 .stamp-bcl-configure: $(TOP)/configure
 	mkdir -p $(TOP)/sdks/builds/bcl
-	cd $(TOP)/sdks/builds/bcl && $(TOP)/configure --with-profile4_x=no $(if $(DISABLE_ANDROID),,--with-monodroid=yes) $(if $(DISABLE_IOS),,--with-monotouch=yes) $(if $(DISABLE_WASM),,--with-wasm=yes) --with-mcs-docs=no --disable-nls --disable-boehm
+	cd $(TOP)/sdks/builds/bcl && $(TOP)/configure --with-profile4_x=yes $(if $(DISABLE_ANDROID),,--with-monodroid=yes) $(if $(DISABLE_IOS),,--with-monotouch=yes) $(if $(DISABLE_WASM),,--with-wasm=yes) --with-mcs-docs=no --disable-nls --disable-boehm 
```

Comment:
`--with-profile4_x` should only be `yes` if `DISABLE_DESKTOP` isn't set

Reply:
Why do you need 4_x profile?

Reply:
Still don't get it. Why don't you build it for monodroid profile

Reply:
Because it needs to run on the host, not on the device.

Reply:
The test suite itself is run on the host, the test app is run on the device. I need profile net_4_x for the `desktop` target that was added.

Reply:
Monodroid does build it's half of this process, it builds dtest-app.